src/Aspire.Dashboard/Otlp/Model/OtlpHelpers.cs

```diff
@@ -44,7 +64,7 @@ public static class OtlpHelpers
 
     public static string FormatTimeStamp(DateTime timestamp)
```

Comment:
Could you add some tests? At least one with invariant culture to double check that milliseconds are successfully added. Test more cultures if it isn't much work.

Reply:
I spent some time trying to add tests here but it's not possible to do that without pessimising the implementation's performance or adding a bunch more layers of indirection. `DateTimeFormatInfo.CurrentInfo.LongTimePattern` and `NumberFormatInfo.CurrentInfo.NumberDecimalSeparator` are queried once, during the static constructor, to produce the pattern during type load. Making it query per-request adds a bunch more string manipulation per formatted value. Adding more indirection is an option, but complicates the code more than I care to.